packages/web/src/components/user-list/UserList.tsx

```diff
@@ -59,18 +56,18 @@ const ConnectedUserList = (props: ConnectedUserListProps) => {
     props.onClickArtistName(handle)
   }
 
-  const { setPageSize, loadMore, pageSize } = props
+  const { loadMore, reset } = props
 
   useEffect(() => {
     // Load initially
-    setPageSize(pageSize)
     loadMore()
-  }, [setPageSize, loadMore, pageSize])
+  }, [loadMore])
 
-  // If page size changes, set it in the store
   useEffect(() => {
-    setPageSize(pageSize)
-  }, [setPageSize, pageSize])
+    return () => {
```

Comment:
This actually produces a type issue because technically the prop returns something
`reset: () => dispatch(reset(ownProps.tag))`

I also kinda like the clarity you get by wrapping the use effect return in an anon function. Also similar to how we agreed to always wrap if statements with `{ }` even if not explicitly necessary

@sliptype had the same comment, but hopefully u guys are cool with pushing this!